### PR TITLE
Fix CNS logging

### PR DESCRIPTION
**Pull request by camrynl**
**Reason for Change**:

 OrchestratorContext cannot always be unmarshalled due to different orchestrator types, so updating CNS logging to exclude it

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests

**Notes**:

### CODE REVIEW

**Review by rbtr — CHANGES_REQUESTED** [review]
if we want to dump (only some) struct fields, maybe `cns.CreateNetworkContainerRequest` should implement [`fmt.Stringer`](https://pkg.go.dev/fmt#Stringer)?

**Review by tamilmani1989 — DISMISSED** [review]
lgtm

File: cns/NetworkContainerContract.go
Comment (rbtr) [inline cns/NetworkContainerContract.go:95]: this log metadata shouldn't be in the String method, it should only return a string representation of the struct
```suggestion
	return fmt.Sprintf("CreateNetworkContainerRequest"+
```

**Review by rbtr — CHANGES_REQUESTED** [review]

**Review by rbtr — APPROVED** [review]
thanks for cleaning this up 🚀

**Comment by rbtr** [comment]
hey @camrynl you are welcome to merge. you should also usually have an "enable auto-merge" button so that you don't have to come back later after the CI and approvals are in 🙂

**Comment by camrynl** [comment]
@rbtr noted, thanks!

**Pull request merged**
